src/openeo_grass_gis_driver/local_collections.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import os
+import json
+from pathlib import Path
+
+from openeo_grass_gis_driver.actinia_processing.actinia_interface import (
+    ActiniaInterface,
+)
+from openeo_grass_gis_driver.actinia_processing.config import Config
+
+
+__license__ = "Apache License, Version 2.0"
+__author__ = "Markus Metz"
+__copyright__ = "Copyright 2018-2021, mundialis"
+__maintainer__ = "mundialis"
+
+
+def get_local_collections():
+    """Get local collections: read all local json files at configured
+    path
+
+    return: collections
+    """
+
+    # get list of all json files at given path
+    local_collections_path = Config.LOCAL_COLLECTIONS
+    jsonfiles = [
+        str(f) for f in Path(local_collections_path).glob("**/*.json") if f.is_file()
+    ]
+
+    collections = {}
+    collections["collections"] = []
+
+    for j in jsonfiles:
+        with open(j) as f:
+            collection = json.load(f)
+            name = j.split("/")[-1][:-5]
+            collection["id"] = "local.mapset.gdallocal.%s" % name
```

Comment:
What is the meaning of the word `mapset` here? And also as a GRASS GIS addon is used, it is not clear, that gdal is used. I would suggest something like `local.folder.rastercube.<name>` or `local.archive.rastercube.<name>` ?

Reply:
Solved by the introduction of the datasource of a collection.